client/src/components/Header.js

```diff
@@ -1,21 +1,22 @@
 import React from "react";
 import AppBar from "@mui/material/AppBar";
 import styled from "styled-components";
-
-const Container = styled.h1`
-  font-size: 1.5em;
-  text-align: center;
-  color: white;
-`;
+import Container from "@mui/material/Container";
 
 function Header() {
   return (
-    <AppBar position="static">
+    <AppBar postion="static">
       <Container maxWidth="xl">
-        <h1>Murial's Restaurant</h1>
+        <h1 style={{ textAlign: "center" }}>Murial's Restaurant</h1>
```

Comment:
Let's talk about utility classes in our next session.  They can be used to avoid online styles entirely.

Reply:
Sounds good